.eslintrc

```diff
@@ -13,6 +14,7 @@
   },
   "rules": {
     "indent": ["error", 2, { "SwitchCase": 1, "MemberExpression": "off" }],
+    "cypress/unsafe-to-chain-command": "off",
```

Comment:
```suggestion
    // TODO: remove after https://github.com/cypress-io/cypress-example-kitchensink/issues/661 has been resolved
    "cypress/unsafe-to-chain-command": "off",
```

Reply:
Thanks for the suggestion! I was also thinking that would be a good idea to add that documentation item.